pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/pscalloydbcluster/create.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: alloydb.cnrm.cloud.google.com/v1beta1
+kind: AlloyDBCluster
+metadata:
+  name: alloydbcluster-${uniqueId}
+spec:
+  displayName: test alloydb cluster
+  location: us-central1
```

Comment:
@199201shubhamsahu do you have any suggestion on what location new AlloyDB test cases should use?

Reply:
@vmiglani Can you use the location "northamerica-northeast1" instead of "us-central1".
This is part of the effort to distribute KCC samples and tests across different regions from the available regions under https://cloud.google.com/alloydb/docs/locations